web/src/pages/requests/index.tsx

```diff
@@ -52,7 +52,9 @@ const Home = () => {
   const search = useSearch<MyLocationGenerics>();
   const navigate = useNavigate<MyLocationGenerics>();
 
-  const { data: rules } = useListUserAccessRules();
+  const { data: rules } = useListUserAccessRules({
```

Comment:
@meyerjrr is this the right API endpoint to be refreshing? I would expect that we need to refresh the `useUserListRequestsUpcoming` call, rather than this one, in order to automatically refresh the request expiry time.

Reply:
Hmm this seemed to solve the issue in the ticket. I will check with Jordi on the implementation here in the frontend

Reply:
Updated @chrnorm, the hooks being passed through seemed to not be working so this has been fixed now.